docs/references/task_library.md

```diff
@@ -1,34 +1,47 @@
 # Task library
 
 ::: viadot.tasks.open_apis.uk_carbon_intensity.StatsToCSV
+
 ::: viadot.tasks.open_apis.uk_carbon_intensity.StatsToExcel
 
 ::: viadot.tasks.azure_data_lake.AzureDataLakeDownload
+
 ::: viadot.tasks.azure_data_lake.AzureDataLakeUpload
+
 ::: viadot.tasks.azure_data_lake.AzureDataLakeToDF
+
 ::: viadot.tasks.azure_data_lake.AzureDataLakeCopy
+
 ::: viadot.tasks.azure_data_lake.AzureDataLakeList
 
 ::: viadot.tasks.azure_key_vault.AzureKeyVaultSecret
+
 ::: viadot.tasks.azure_key_vault.CreateAzureKeyVaultSecret
+
 ::: viadot.tasks.azure_key_vault.DeleteAzureKeyVaultSecret
 
 ::: viadot.tasks.azure_sql.AzureSQLBulkInsert
+
 ::: viadot.tasks.azure_sql.AzureSQLCreateTable
+
 ::: viadot.tasks.azure_sql.AzureSQLDBQuery
 
 ::: viadot.tasks.bcp.BCPTask
 
 ::: viadot.tasks.great_expectations.RunGreatExpectationsValidation
 
 ::: viadot.tasks.sqlite.SQLiteInsert
+
 ::: viadot.tasks.sqlite.SQLiteSQLtoDF
 
 ::: viadot.tasks.supermetrics.SupermetricsToCSV
+
 ::: viadot.tasks.supermetrics.SupermetricsToDF
 
 :::viadot.task_utils.add_ingestion_metadata_task
+
 :::viadot.task_utils.get_latest_timestamp_file_path
 
 ::: viadot.tasks.cloud_for_customers.C4CToDF
+
 ::: viadot.tasks.cloud_for_customers.C4CReportToDF
```

Comment:
I don't understand why you ungrouped these. It seemed cleaner to me when these were grouped.